label_studio/tests/webhooks/organizations.tavern.yml

```diff
@@ -57,6 +57,17 @@ stages:
 - id: get_user_token
   type: ref
 
+# Test this functionality with legacy tokens, need to enable them for the org
+- name: enable_legacy_api_tokens
+  request:
+    url: "{django_live_url}/api/current-user/whoami"
+    method: GET
+  response:
+    status_code: 200
+    verify_response_with:
```

Comment:
why are we using it this way..? enabling something for the org, through a response verify function?? Why cant we enable it with a normal API call (maybe there isnt one)? This seems confusing and will be missed if anyone has to figure out whats going on

Reply:
Ok yeah, fair enough, I'll change that